### PR TITLE
Adds linting for the rest of the JS codebase and fixes all errors.

### PR DESCRIPTION
**Pull request by jessex**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "dev": "nodemon ./bin/www",
     "clean": "shx rm -rf ./build",
     "build": "npm run clean && cross-env NODE_ENV=\"production\" webpack",
-    "test": "eslint src"
+    "test": "eslint routes src views webpack app.js"
```
Comment (jessex) [inline package.json]: This is the operative change.

File: routes/auth.js [hunk 2]
```diff
@@ -15,34 +15,40 @@
...
 // =============================================================================
 
-var express = require('express');
-var router = express.Router();
-var passport = require('passport');
-var dotenv = require('dotenv');
-var util = require('util');
-var querystring = require('querystring');
+const express = require('express');
+
```
Comment (jjdressel11) [inline routes/auth.js]: Is this newline intentional?

File: routes/index.js [hunk 3]
```diff
@@ -15,13 +15,13 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
 
-var express = require('express');
-var secured = require('../src/middleware/secured');
-const { join } = require('path');
-var router = express.Router();
+const express = require('express');
+const secured = require('../src/middleware/secured');
+
```
Comment (jjdressel11) [inline routes/index.js]: Intentional newline?

Comment (jessex) [reply routes/index.js]: Generated by the `eslint --fix` command, for some reason. Will remove.

File: routes/auth.js (hunk 2 above)
Comment (jessex) [reply routes/auth.js]: Will remove, see below comment.

File: routes/index.js (hunk 3 above)
Comment (jessex) [reply routes/index.js]: Actually required in this case. There must be a newline between a `require` and a non-require.

**Pull request merged**